examples/tests/design.test.ts

```diff
@@ -0,0 +1,56 @@
+// design test for material.io
+
+import { shortest } from "@antiwork/shortest";
+
+// Material Design 3 color system
+shortest(
+  "Visit material.io and verify the Material Design 3 color system implementation. Primary color should be #6750A4 (purple) with tonal palette ranging from 0-100. Check dynamic color theming in light and dark modes. Verify key color roles: primary, secondary (#9C27B0), tertiary (#FFB4AB), and error (#B3261E) across components",
+  {
+    baseColors: {
+      primary: "#6750A4",
+      secondary: "#9C27B0",
```

Comment:
Seems repetitive with above?

Reply:
Made the test cases in this file less repetitive.